roles/nuage_node/vars/main.yaml

```diff
@@ -6,4 +6,10 @@ client_cert: "{{ vsp_openshift_dir }}/nuage.crt"
 client_key: "{{ vsp_openshift_dir }}/nuage.key"
 ca_cert: "{{ vsp_openshift_dir }}/ca.crt"
 api_server: "{{ openshift_node_master_api_url }}"
+nuage_mon_rest_server_port: "{{ nuage_openshift_monitor_rest_server_port | default('8444') }}"
```

Comment:
8444 is the same port that the controllers service listens on, since the nuage mon service and the controllers service will be hosted together on the master hosts, the nuage mon service should default to a different port.